### DISCUSSION TITLE
Attributes Mapping via API

### DISCUSSION THREAD

**Discussion by NOPKGer** [cwapi3d/cwapi3dpython#194] [Q&A]
Hey, 
Is it possible to map converted IFC objects via the API
or is it only possible through the built in mapper?
I need the attributes from a specific set but have not found a way to access them with the API.

**Comment by Brunner246** [comment]
Hi @NOPKGer 
This is not possible at the moment. Currently, the mapping must be done manually.

**Comment by NOPKGer** [comment]
hey @Brunner246 thanks for the quick reply. 
if you say not at the moment then is it planned to come?

**Reply by Brunner246** [reply]
I have added it into our backlog. We will check if and how we could provide that via cwapi3d.
